docs/install-1.6.md

```diff
@@ -79,6 +79,10 @@ newly-installed executable.
 
 # Step 2 - Installing the Service Catalog
 
+It is recommended to deploy the service catalog with your local copy of the repo
```

Comment:
What's the functional difference between installing the latest tagged image in Quay with installing from the latest tagged local copy?

Reply:
If I understand correctly, you're asking "what's the difference between pointing at the v0.0.16 images from quay, vs using images built yourself from the v0.0.16 state of the repo"?

In that case, there is no functional difference; the images are identical. But this PR is trying to address a different issue: if you've checked out the repo at any other time other than right when a new version was cut, by running `helm install` without explicitly providing the images that match your currently checked out repo, your local charts and walkthrough files you're using have a chance of being out of sync with the images that are by default installed (the latest versioned release).

The solution here is to make it such that your repo matches the images you will install. If you want to use the default images, then, you have to have your repo checked out at the latest versioned release as well. If you want to use images for whatever state your repo is in currently, you have to source custom images, either from quay.io or by building them yourself.

Does changing `latest tagged version` to `latest versioned release (for example, v0.0.16)` here clarify?

Reply:
I put this recommendation line here because I think it's an easier process for the new user to just make sure their repo's at the latest versioned release than to have to build and/or reference the explicit images.